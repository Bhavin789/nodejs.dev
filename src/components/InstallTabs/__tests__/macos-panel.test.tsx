/* eslint-disable @typescript-eslint/explicit-function-return-type */
import React from 'react';
import renderer from 'react-test-renderer';
import MacOSPanel from '../MacOSPanel';

describe('Tests for MacOSPanel component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<MacOSPanel />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
