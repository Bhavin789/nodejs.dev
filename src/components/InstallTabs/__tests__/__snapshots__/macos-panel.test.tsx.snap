// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for MacOSPanel component renders correctly 1`] = `
<div>
  <pre
    className="shell-box"
  >
    <div
      className="shell-box-top"
    >
      <span>
        SHELL
      </span>
      <button
        onClick={[Function]}
        type="button"
      >
        copy
      </button>
    </div>
    <code>
      <span
        className="install__text__no-select"
      >
        $
      </span>
      <span
        className="install__text__command"
      >
        curl -o- 
      </span>
      https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh
      <span
        className="install__text__command"
      >
         | bash 
      </span>
    </code>
  </pre>
  <pre
    className="shell-box"
  >
    <div
      className="shell-box-top"
    >
      <span>
        SHELL
      </span>
      <button
        onClick={[Function]}
        type="button"
      >
        copy
      </button>
    </div>
    <code>
      <span
        className="install__text__no-select"
      >
        $
      </span>
      <span
        className="install__text__command"
      >
        nvm 
      </span>
      install --lts
    </code>
  </pre>
  <br />
  <br />
  <a
    className="install__docs-button"
    href="https://nodejs.org/en/download/package-manager/#nvm"
    rel="noopener noreferrer"
    target="_blank"
  >
    Read documentation
  </a>
</div>
`;
